### PR TITLE
More graffiti

### PR DESCRIPTION
**Pull request by unid15**
-Allows choosing which graffiti you will draw

-Adds more graffiti
 -4 directional arrows
 -heart
 -"borgs rogue" text
 -"shitcurity" text
 -"catbeast here" text
 -"vox are pox" text

-3 chinese hieroglyphs
-"securites eunt domus" text
-2 syndicate logos
-NT logo
-lambda
-chaos undivided logo
-50 blessings logo

-3 unique crude dick drawings (only available with severe brain damage)
-"muh valids" text (only available with severe brain damage)

### CODE REVIEW

**Comment by PJB3005** [comment]
:-1: :x: :100: 

VOX NOT POX

**Comment by DrCelt** [comment]
>no "securites eunt domus"

Go home.

**Comment by Unfit** [comment]
the mime's crayon makes it look like "securites cunt domus" through

File: code/game/objects/items/crayons.dm [hunk 1]
```diff
@@ -69,11 +69,37 @@
...
+					"Uboa"="uboa",
+					"Rogue cyborgs"="borgsrogue",
+					"Shitcurity"="shitcurity",
+					"Catbeast here"="catbeast",
+					"Vox are pox"="voxpox"
+					//"Hieroglyphs"="hieroglyphs[rand(1,3)]"
+				)
+				preference = input("Choose the graffiti") in graffitis
+				drawtype=graffitis[preference]
+				if(drawtype=="random") drawtype="graffiti"
```
Comment (ComicIronic) [inline code/game/objects/items/crayons.dm]: Why not just make the input "Random"="graffiti"

**Comment by Unfit** [comment]
adding ability to write custom messages donut merge

**Comment by DrCelt** [comment]
custom messages could use the maptext var

**Comment by Unfit** [comment]
actually i'll do custom messages in another PR


added lambda, chaos undivided, 50 blessings to the list of graffitis
those with more than 60 brain damage have their own separate list of things they can draw

**Comment by Probe1** [comment]
nice!

**Comment by CptWad** [comment]
:+1:

File: code/game/objects/effects/decals/crayon.dm [hunk 2]
```diff
@@ -5,10 +5,12 @@
 	layer = 2.1
 	anchored = 1
 
-/obj/effect/decal/cleanable/crayon/New(location,main = "#FFFFFF",shade = "#000000",var/type = "rune")
+/obj/effect/decal/cleanable/crayon/New(location,main,shade,var/type="rune")
 	..()
 	loc = location
 
+	if(!main || !shade || !type) return
```
Comment (ComicIronic) [inline code/game/objects/effects/decals/crayon.dm]: Why would you do this and remove the default values, that's why the default values exist

Comment (Unfit) [inline code/game/objects/effects/decals/crayon.dm]: oh shit I accidentally did that while working on custom text and forgot to remove it later

**Comment by FudgePucker** [comment]
What about the clownkart?
Can it draw dicks everywhere if a coin is entered?

**Comment by Unfit** [comment]
Not really because dicks are only available when you use a crayon on the ground with more than 60 brain damage
Drawing random graffiti won't ever draw dicks

**Comment by KasparoVy** [comment]
VOX NOT POX

**Comment by CptWad** [comment]
thank you chicken now I can do the gimmick I have been waiting for

**Pull request merged**